tests/requirements.in

```
@@ -0,0 +1,12 @@
+pytest
+pytest-cov
+coverage
+pytest-mock
+codecov
+radon
+Flask_Cors
+psycopg2-binary
+sentry_sdk
+pydocstyle
+f8a_worker @ git+https://github.com/fabric8-analytics/fabric8-analytics-worker.git@066c2f6#egg=f8a_worker
+fabric8a_auth @ git+https://github.com/fabric8-analytics/fabric8-analytics-auth.git@5ff9438#egg=fabric8a_auth
```

Include main requirements.in instead of repeating the dependencies here.
```suggestion
pytest
pytest-cov
coverage
pytest-mock
codecov
radon
pydocstyle
-r ../requirements.in
```